src/EditorFeatures/CSharpTest2/Recommendations/WhenKeywordRecommenderTests.cs

```diff
@@ -425,19 +425,46 @@ class C
     void M() { switch (new object()) { case var $$ when } }
 }");
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/30848"), Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestForSwitchCase_SemanticCheck_NotAfterTypeAliasVar() =>
```

Comment:
1. methods with => have the=> on hte next line in IDE code.
2. IDE test methods don't use =>, they use normal blocks. 

Could you follow the patterns of the rest of the IDE code here?  We can certainly consider a PR in the future to update tests to be of this form if people liek it.

Reply:
➡️ Fixed in 7b1c2a6028ff3423790e02d7001a87ed52e349d7